test/parallel/test-tls-alert.js

```diff
@@ -32,15 +31,17 @@ if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-const tls = require('tls');
 
+const assert = require('assert');
+const { spawn } = require('child_process');
```

Comment:
FWIW, I think module name sorting would be more consistent here, as we can use many variable names in destructuring and only one module name. [See examples](https://github.com/nodejs/node/search?utf8=%E2%9C%93&q=%22require%28%27..%2Fcommon%2Fwpt%27%29%22&type=).